orbit/cmd/desktop/desktop.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	_ "embed"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/fleetdm/fleet/v4/pkg/open"
+	"github.com/getlantern/systray"
+)
+
+//go:embed favicon.ico
+var icoBytes []byte
```

Comment:
We will probably want a handful of different icons, not sure if it's worth while to think about this at this point, but eventually we'll use it to show different states such as policy not passing, or update needed, etc.